README.md

```diff
@@ -1,4 +1,4 @@
-Jena README
+Jena README 
```

Comment:
Extra space?

Reply:
It is unintentional. A colleague was testing some github logic and apparently selected the jena fork's README.md. I've committed a repaired README.md

Reply:
And it's gone.